### PR TITLE
Adds pizza day redirect

### PR DESCRIPTION
**Pull request by paulovictor-cb**
## What changed? Why?

This PR adds a redirect for pizza day deep link

## How has it been tested?

- [X] Locally
- [ ] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

### CODE REVIEW

**Review by travisbloom-cb — APPROVED** [review]

File: next.config.js [hunk 1]
```diff
@@ -253,6 +253,12 @@ const nextConfig = {
         destination: '/races/:paths*',
         permanent: true,
       },
+      // tweet at person campaigns
+      {
+        source: '/pizza',
+        destination: '/action/tweet-at-person/2024_05_22_PIZZA_DAY',
+        permanent: false,
```
Comment (lucasrmp-cb) [inline next.config.js:260]: This should be permanent, no?

Comment (travisbloom-cb) [reply next.config.js:260]: na, vanity urls are always temp so they can be reused later on if needed

Comment (travisbloom-cb) [reply next.config.js:260]: open to changing but that's what i was assuming

Comment (paulovictor-cb) [reply next.config.js:260]: https://nextjs.org/docs/app/api-reference/next-config-js/redirects

permanent true or false - if true will use the 308 status code which instructs clients/search engines to cache the redirect forever, if false will use the 307 status code which is temporary and is not cached.

**Pull request merged**